include/itkMorphologicalContourInterpolator.hxx

```diff
@@ -26,7 +26,6 @@
 #include "itkImageAlgorithm.h"
 #include "itkImageRegionConstIteratorWithIndex.h"
 #include "itkImageRegionIterator.h"
-#include "itkMorphologicalContourInterpolator.h"
```

Comment:
Declaration of `SegmentBetweenTwo` at line 46 should be moved behind class declaration at line 57. Take a look at this failure: https://open.cdash.org/viewBuildError.php?buildid=7630953.